wesnoth-tsg.po

```diff
@@ -54,10 +55,8 @@ msgstr ""
 
 #. [campaign]: id=The_South_Guard
 #: data/campaigns/The_South_Guard/_main.cfg:22
-#, fuzzy
-#| msgid "(Rookie level, 9 scenarios.)"
 msgid "(Novice level, 8 scenarios.)"
-msgstr "(Úroveň zelenáč, 9 scénářů.)"
+msgstr "(Úroveň zelenáč, 8 scénářů.)"
```

Comment:
pokud to dobře chápu, tak hra pořád rozlišuje rookie a novice (rookie je ještě jednodušší)
```suggestion
msgstr "(Úroveň nováček, 8 scénářů.)"
```

Reply:
Asi ano, chtělo by to aktualizovat v konvenci.